tests/Collections/CachedConfigCollectionTest.php

```diff
@@ -9,53 +9,47 @@
 use PHPUnit\Framework\TestCase;
 use Prophecy\Prophet;
 use SilverStripe\Config\Collections\ConfigCollectionInterface;
+use SilverStripe\Core\Config\Config;
```

Comment:
Holy coupling, Batman!

We should not be creating interdependencies like this - they're a pollutant.

Reply:
🤦 This import doesn't even do anything, I'm guessing I typo'd while I was writing `ConfigCollectionInterface` and my IDE helpfully autocompleted this for me. At least it served as a prompt to clean up the leftover Prophecy imports.